src/index.spec.ts

```diff
@@ -43,11 +46,8 @@ describe("json-rpc-e2e", () => {
         params: { data: "hi" }
       }
     };
-    return request(server.server)
-      .post("/rpc/v1")
-      .send({ method: "test.testError", params: { data: "hi" } })
-      .expect(403)
-      .expect(errorObj);
+    const resp = service.testError({ data: "hi" });
+    return expect(resp).rejects.toThrowError();
```

Comment:
Is there a way to specify that the error should have a certain code/error/message here?